weblink/Weblink.hx

```diff
@@ -132,4 +135,9 @@ class Weblink {
 		var route = this.routes[request.path];
 		route.get("HEAD")[0](request, response);
 	}
+
+	public function set_fnRouteNotFound(value:Func):Func {
```

Comment:
Can you change the name? the "set_fn" prefix is not intuitive. What about ``pathNotFound``?

Reply:
Just made a commit regarding the naming of `fnRouteNotFound` I changed it to simply be `pathNotFound` and some additional documenation regarding the anyonmous function in general